google/generativeai/types/content_types.py

```diff
@@ -392,12 +388,41 @@ def from_proto(cls, proto) -> FunctionDeclaration:
     def to_proto(self) -> glm.FunctionDeclaration:
         return self._proto
 
+    @classmethod
+    def from_function(
+        cls, function: Callable[..., Any], descriptions: dict[str, Any] | None = None
```

Comment:
```suggestion
        cls, function: Callable[..., Any], descriptions: dict[str, str] | None = None
```

This matches `_generate_schema`

Reply:
Since this is a nit I just pushed directly.